datalad_hirni/commands/dicom2spec.py

```diff
@@ -3,7 +3,7 @@
 """
 
 import logging
-from os.path import exists
+from os.path import exists, relpath
```

Comment:
Let's converge on `import os.path as op` and not do multiple imports per line, they are a pain to clean up later on